Resolve the UX issues of the registration page - Closes #2465

colado opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### What issue have I solved?
<!--- Complementary description if needed -->
#2465

### How have I implemented/fixed it?
- Implemented all changes on [new registration screens](https://projects.invisionapp.com/d/main?origin=v7#/console/18244706/380009763/preview), except for multiStep, which will be implemented on a different issue.
- The progress bar and multiStep component are being implemented on #2578. This will replace the steps on top of each screen.

### How has this been tested?
Made sure registration and passphrase generation was done successfully and UI aligns with new design.


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

osvaldovega commented on src/components/shared/passphraseBackup/index.js:
```diff
@@ -42,55 +49,28 @@ class PassphraseBackup extends React.Component {
-                </Tooltip>
-              </h2>
+              <h2>{t('Paper wallet')}</h2>
+              <p className={styles.infoText}>{t('You can also download, print and store safely your pasphrase.')}</p>
```
typo 
```suggestion
              <p className={styles.infoText}>{t('You can also download, print and store safely your passphrase.')}</p>
```

colado replied:
✅

osvaldovega commented on src/components/shared/passphraseRenderer/index.js:
```diff
@@ -0,0 +1,101 @@
+    if (missingWords.includes(i) && fieldSelected !== i) return styles.emptyInput;
+    if (isConfirmation) return styles.disabled;
+    return styles.default;
+  }
```
we need to improve this function to remove eslint complexity.

colado replied:
I have moved some of that logic to other functions. Let me know if that looks ok to you.

colado commented on src/components/screens/register/accountCreated.test.js:
```diff
@@ -1,33 +1,19 @@
-
   const props = {
-    account,
+    t: jest.fn(),
```
✅

colado commented:
Thank you, Osvaldo. I have implemented also the changes on the visual issues. Let me know if something else needs changing or if there's something I missed.

slaweet commented on src/components/shared/passphraseRenderer/passphraseRenderer.css:
```diff
@@ -0,0 +1,131 @@
+      font-weight: var(--font-weight-bold);
+      color: var(--color-maastricht-blue);
+    }
+  }
```
💅 Doesn't look very DRY.
```css
  & .selected,
  & .selected::before {
    font-weight: var(--font-weight-bold);
    color: var(--color-maastricht-blue);
  }
```
would be DRYer.

colado replied:
✅

slaweet commented on src/components/shared/passphraseRenderer/passphraseRenderer.css:
```diff
@@ -0,0 +1,131 @@
+      font-weight: var(--font-weight-bold);
+      color: var(--color-burnt-sienna);
+    }
+  }
```
💅 Doesn't look very DRY.

colado replied:
✅

slaweet commented on src/components/shared/passphraseRenderer/passphraseRenderer.css:
```diff
@@ -0,0 +1,131 @@
+      font-weight: var(--font-weight-bold);
+      color: var(--color-ufo-green);
+    }
+  }
```
💅 Doesn't look very DRY.

colado replied:
✅

slaweet commented on src/components/shared/passphraseRenderer/passphraseRenderer.css:
```diff
@@ -0,0 +1,131 @@
+      font-weight: var(--font-weight-bold);
+      color: var(--color-ultramarine-blue);
+    }
+  }
```
💅 Doesn't look very DRY. 
IMO would be more readable to combine common properties of `.emptyInput` `.emptyInputFocused`  `.emptyInput::before ` `.emptyInputFocused::before ` where possible.

Maybe separating `emptyInputFocused` into two clases `.emptyInput.focused` would make it easier not to repeat which properties are applied because the input is empty and which ones because it's focused.

slaweet replied:
Also calling it `input` confused me at first. I wanted to recommend https://www.w3schools.com/cssref/sel_empty.asp and https://www.w3schools.com/cssref/sel_focus.asp instead of some classes.

Then I saw it's just a span with a word, so `emptyWord` would be IMO more descriptive class name.

slaweet commented on src/components/shared/passphraseRenderer/passphraseRenderer.css:
```diff
@@ -0,0 +1,123 @@
+    &::before {
+      color: var(--color-ultramarine-blue);
+    }
+  }
```
Thank you, Sergio. This file looks much cleaner now.

Just one thing left. 💅 This one still repeats the same color and could be reduced to:
```css
  & .emptyWordFocused,
  & .emptyWordFocused::before {
    color: var(--color-ultramarine-blue);
  }
```

osvaldovega commented:
Hey Sergio I just checked the code related to code all good now the design, the interface looks much better but there are some missing things.

Here are.

## Choose your avatar
🎨Choose your avatar screen has bigger container for the avatar for what looks like there is a lot of space between each avatar.
🎨The selected avatar looks bigger now.
🎨This is the same for all the pages but the primary button is smaller than the design.
<img width="1160" alt="Screen Shot 2019-10-25 at 15 07 20" src="https://user-images.githubusercontent.com/11184770/67574790-67e98880-f73b-11e9-9626-0238c5418dfb.png">

## Save your passphrase
🎨For some reason the content looks align more to the left than the design so all the elements in this page looks bigger than the design.
🎨Same as previous page the primary button is smaller than the design, should be bigger.
🎨check the margin/padding from the all the elements looks like are not the same.
<img width="888" alt="Screen Shot 2019-10-25 at 15 13 33" src="https://user-images.githubusercontent.com/11184770/67575016-eb0ade80-f73b-11e9-96a8-38683690d177.png">

## Confirm your passphrase
🎨Has the same error as the previous screens, everything looks like is align to the left (Think is a common error), if you fix it for the first page probably will be fix for all the other screens.
🎨The primary button too, is smaller than design.
<img width="1120" alt="Screen Shot 2019-10-25 at 15 19 40" src="https://user-images.githubusercontent.com/11184770/67575126-2b6a5c80-f73c-11e9-8aae-09488dbd7f3c.png">

colado commented:
Thank you, @osvaldovega as we discussed, I fixed the button width

osvaldovega commented:
All good @colado most of the design now is fixed there are some minor things that I think are good so we can let @imdominic check it and let us know what he thinks about that.
@Efefefef most of the things are done, please check it and see if Dom can review it to get his feedback

colado commented:
Thank you, @Efefefef, I have changed the paper wallet name and displayed that text in one line. The progress bar is being implemented in #2578. I have updated the description of the PR for clarification.

imdominic commented:
As Andrei mentioned above, the steps above are not showing up, but Sergio told me that they are gonna be involved in a separate pull request.
Would be ideal to increase the height of the box slightly, as per design, in order to make the section less cramped.

In terms of UX:

- 1st word should be already active and you should have the options there, once you click "I wrote it down"
- You should be able to change the words before you hit confirm. So the options should show up if you click on the word selected in black.

Rest looks really good, thanks Sergio :)
